src/rfq/protocols/bebop/client.rs

```diff
@@ -83,28 +83,28 @@ pub struct BebopClient {
 impl BebopClient {
     pub fn new(
         chain: Chain,
-        pairs: HashSet<(String, String)>,
+        tokens: HashSet<String>,
         tvl: f64,
         ws_user: String,
         ws_key: String,
         quote_tokens: HashSet<String>,
     ) -> Result<Self, RFQError> {
         let url = chain_to_bebop_url(chain)?;
 
-        let mut pair_names: HashSet<String> = HashSet::new();
-        for pair in pairs {
-            pair_names.insert(pair_to_bebop_format(&pair)?);
-        }
         let mut quote_tokens_checksummed: HashSet<String> = HashSet::new();
+        let mut tokens_checksummed: HashSet<String> = HashSet::new();
 
         for token in quote_tokens.iter() {
             quote_tokens_checksummed.insert(checksum(token)?.to_string());
         }
+        for token in tokens.iter() {
+            tokens_checksummed.insert(checksum(token)?.to_string());
+        }
```

Comment:
why don't we force the user to insert the token addresses as bytes already? then we don't need to worry about this